charts/pelorus/templates/prometheus-scrape-config-secret.yaml

```diff
@@ -19,14 +19,26 @@
         labels:
           federated_job: "federated-prometheus-{{ .id }}"
   {{- end }}
+  {{- range .Values.external_prometheus_hosts }}
+  - job_name: "external-prometheus-{{ .id }}"
+    # TODO any other config? I believe can be anything from here https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config
```

Comment:
lets talk about this before merging

Reply:
this was already discussed. no other config atm.